### DISCUSSION TITLE
Windows Package Manager 1.3.432-preview

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1938] [Releases]
This release is the first development build after the Windows Package Manager 1.2 release candidate build for Windows 10 (1809+).

Experimental features have been enabled in this release. This build will be released to Windows Insider Dev builds, and Windows Package Manager Insiders.

## What's Changed

* Add Package Dependencies to index.  by @hackean-msft in https://github.com/microsoft/winget-cli/pull/1667
* Update windows-package-manager-release-roadmap.md by @opsetech in https://github.com/microsoft/winget-cli/pull/1770
* Bug fix: PackageTrackingCatalog and SQLiteIndexUpdate by @hackean-msft in https://github.com/microsoft/winget-cli/pull/1780
* Update TSG with more information about a known issue by @ashpatil-msft in https://github.com/microsoft/winget-cli/pull/1785
* Added argument to control whether to upgrade packages if they have "unknown" versions by @jedieaston in https://github.com/microsoft/winget-cli/pull/1765
* First drafted client cmdlets from Hackathon 221 by @denelon in https://github.com/microsoft/winget-cli/pull/1760
* Fix some build warnings by @lechacon in https://github.com/microsoft/winget-cli/pull/1794
* Fix Typo in upgrade.md by @floh96 in https://github.com/microsoft/winget-cli/pull/1822
* Add support for UnsupportedOSArchitectures manifest element by @lechacon in https://github.com/microsoft/winget-cli/pull/1807
* Added check for maximum size of downloaded file names by @jedieaston in https://github.com/microsoft/winget-cli/pull/1842
* Fix DLL load error in WinGetUtil by @lechacon in https://github.com/microsoft/winget-cli/pull/1844
* Update localization strings by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1845
* Add comments for localization by @lechacon in https://github.com/microsoft/winget-cli/pull/1847
* Moved "Installing Dependencies" message to only print if there are dependencies to install. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1851
* Adds experiment to know if PATH is common issue. by @Lewiscowles1986 in https://github.com/microsoft/winget-cli/pull/1841
* Check FS feature flags instead of checking for NTFS by @sredna in https://github.com/microsoft/winget-cli/pull/1859
* Add titles to 1.1 schema for ExpectedReturnCode and UnsupportedOSArchitecture by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/1862
* Split pipeline build job into x86 and x64 by @lechacon in https://github.com/microsoft/winget-cli/pull/1852
* Allow upgrades in packages that register a different installer type by @lechacon in https://github.com/microsoft/winget-cli/pull/1796
* Add a default user agent to REST source calls by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1839
* Add upgrade functionality in Com api by @yao-msft in https://github.com/microsoft/winget-cli/pull/1853
* Print the upgrade table during `upgrade --all` by @jedieaston in https://github.com/microsoft/winget-cli/pull/1866
* Add support for markets by @lechacon in https://github.com/microsoft/winget-cli/pull/1806
* [ImgBot] Optimize images by @vedantmgoyal2009 in https://github.com/microsoft/winget-cli/pull/1871
* Fix crash that can occur when failure pointers are null by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1880
* Added extra check for valid arguments in `upgrade`. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1874
* Add InstallerErrorCode to COM interface by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1926
* Update gif animation for winget install wingetcreate by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/1931
* Bump version to 1.3 by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1933

## New Contributors
* @opsetech made their first contribution in https://github.com/microsoft/winget-cli/pull/1770
* @floh96 made their first contribution in https://github.com/microsoft/winget-cli/pull/1822
* @Lewiscowles1986 made their first contribution in https://github.com/microsoft/winget-cli/pull/1841
* @sredna made their first contribution in https://github.com/microsoft/winget-cli/pull/1859

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.0.12576...v1.3.431

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.3.431'>Windows Package Manager 1.3.432-preview</a>.</em>

**Comment by 0Grit** [comment]
@denelon I'd like to be able to run the installer from within a windows sandbox, which is not recognizing the msixbundle.
Could a .exe installer be released sustainably?

**Reply by 0Grit** [reply]
Disregard `Add-AppxPackage` worked fine to install it and [VC++ v14 Desktop Framework Package](https://docs.microsoft.com/troubleshoot/cpp/c-runtime-packages-desktop-bridge#how-to-install-and-update-desktop-framework-packages)
